templates/index.html

```diff
@@ -227,6 +222,11 @@ <h3 class="section-head">Mental State Graph</h3>
                     success: function (data) {
                         data = data[0];
                         console.log(data);
+                        let date = data.timestamp;
+                        let emotion = data.emotion;
+                        let confidence = data.confidence;
+                        $('#log_section').append('<p> At ' + formatTime(date) + ", the patient is " + emotion + ", with confidence " + confidence + '<p>');
```

Comment:
nit: personally find the format `<time> - patient is <emotion> with confidence <confidence>` more concise.

Reply:
Resolved